config/vufind/config.ini

```diff
@@ -220,6 +220,7 @@ session_name = VUFIND_SESSION
 ;   - Innovative (for INNOPAC; see also Sierra/SierraRest)
 ;   - Koha (basic database access only)
 ;   - KohaILSDI (more features via ILS-DI API)
+;   - KohaRest (more features via native API)
```

Comment:
Should we say something about minimum required Koha version here? Should we consider deprecating either of the other drivers?